python/cudf/cudf/_lib/column.pyx

```diff
@@ -599,7 +600,7 @@ cdef class Column:
                     mask = as_buffer(
                         rmm.DeviceBuffer(
                             ptr=mask_ptr,
-                            size=bitmask_allocation_size_bytes(size+offset)
+                            size=bitmask_allocation_size_bytes(base_size)
```

Comment:
This is fine because if `column_owner` is None then `base_size == size + offset`.